lxd/storage.go

```diff
@@ -55,6 +56,7 @@ func resetContainerDiskIdmap(container instance.Container, srcIdmap *idmap.Idmap
 			jsonIdmap = "[]"
 		}
 
+		logger.Debug("Setting new volatile.last_state.idmap from source instance", log.Ctx{"project": container.Project(), "instance": container.Name(), "sourceIdmap": srcIdmap})
```

Comment:
@stgraber I don't really understand the reason for this function. In remote copies lxc command strips `volatile.last_state.idmap` from the config sent for the new instance (unless `--refresh` is set), but then this function copies the original value from the source instance's config which is sent unmodified in the migration stream and re-applied if the new container's idmap is empty or different.

The accompanying comment to the caller in migrationSink is: 

https://github.com/lxc/lxd/blob/c6037ded3cd2c59692a32efc7a0790e6db44d8d1/lxd/migrate_instance.go#L1057-L1061